main.py

```diff
@@ -95,6 +99,7 @@ def __call__ (self, parser, namespace, values, option_string = None):
     parser.add_argument('--default_theta', type=float, default=90, help="radius for the default view")
     parser.add_argument('--default_phi', type=float, default=0, help="radius for the default view")
     parser.add_argument('--default_fovy', type=float, default=20, help="fovy for the default view")
+    parser.add_argument('--default_img_w', type=float, default=1, help="zero123 weight for the default view")
```

Comment:
Not sure if that's really required. If there's just 1 reference view, why would you ever set this to a value other than 1?